coredns/tox.ini

```diff
@@ -0,0 +1,37 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist =
+    coredns
+    docker
+    flake8
+
+[testenv]
+platform = linux|darwin
+passenv =
+    DOCKER*
+    COMPOSE*
+
+[common]
+deps =
+    ../datadog_checks_base
```

Comment:
You should be able to replace both of those lines with `-e../datadog_checks_base[deps]`